AsyncFunctions/asyncfunctions.js

```diff
@@ -0,0 +1,35 @@
+functions = [
+  () => new Promise((resolve) => setTimeout(() => resolve(4), 5000)),
+  () => new Promise((resolve) => setTimeout(() => resolve(10), 15000)),
+  () => new Promise((resolve) => setTimeout(() => resolve(16), 10000)),
+];
+
+const promiseAll = (functions) => {
+  return new Promise((resolve, reject) => {
+    var resolved_arr = [];
+    var count = 0;
+
+    const checkCount = () => {
+      if (count === functions.length) {
+        resolve(resolved_arr);
+      }
+    };
+
+    
+
```

Comment:
```suggestion
```
do not use multiple empty lines, only use single empty line for spacing purpose. using multiple line unnecessarily will increase LOC only.